selfdrive/car/mazda/carstate.py

```diff
@@ -15,7 +15,7 @@ def __init__(self, CP):
     self.crz_btns_counter = 0
     self.acc_active_last = False
     self.low_speed_alert = False
-    self.lkas_allowed_speed = False
+    self.lkas_allowed_speed = CP.minSteerSpeed <= 0
```

Comment:
Not sure this will do it. `minSteerSpeed` is set to 2 by default by OP.  Change it to this:
```
CP.minSteerSpeed <= 2
```

Reply:
Where do you see that? It's set in interfaces here: https://github.com/commaai/openpilot/blob/247b66f4a801d3621575edf5a247c54e275d2826/selfdrive/car/interfaces.py#L80

Reply:
I could be wrong, I haven't been able to drive/test the last couple of months!. 

Last time a checked: 
https://github.com/commaai/openpilot/blob/master/selfdrive/car/mazda/interface.py#L62

I did notice Mazda CX5 2022 min steer speed was coming at 2.  I didn't check well, but I assumed OP doesn't want to steer below 2 but I could be wrong. 

@adeebshihadeh ?

Reply:
This is right, but this line is confusing. @sshane let's revert back to setting this in the else block in update()